nix/install.go

```diff
@@ -43,3 +48,20 @@ func Install() error {
 
 	return errors.WithStack(cmd.Wait())
 }
+
+func EnsureInstalled(cmd *cobra.Command, args []string) error {
+	_, err := exec.LookPath("nix-shell")
```

Comment:
Do you think we should also check to see if `/nix` exists? That way we don't try to install it again if the user forgot to restart their session or a macOS update stomped on `/etc/zshrc`.

Reply:
oh good call